application/config/config.go

```diff
@@ -54,14 +55,15 @@ import (
 )
 
 const (
-	deeproxyApiUrlKey     = "DEEPROXY_API_URL"
-	FormatHtml            = "html"
-	FormatMd              = "md"
-	snykCodeTimeoutKey    = "SNYK_CODE_TIMEOUT" // timeout as duration (number + unit), e.g. 10m
-	DefaultSnykApiUrl     = "https://snyk.io/api"
-	DefaultDeeproxyApiUrl = "https://deeproxy.snyk.io"
-	pathListSeparator     = string(os.PathListSeparator)
-	windows               = "windows"
+	deeproxyApiUrlKey        = "DEEPROXY_API_URL"
+	FormatHtml               = "html"
+	FormatMd                 = "md"
+	snykCodeTimeoutKey       = "SNYK_CODE_TIMEOUT" // timeout as duration (number + unit), e.g. 10m
+	DefaultSnykApiUrl        = "https://snyk.io/api"
+	DefaultDeeproxyApiUrl    = "https://deeproxy.snyk.io"
+	DefaultSnykCodeApiGovUrl = "https://api.snyk.io"
```

Comment:
Question: This seems redundant to `DefaultSnykApiUrl`, do we really need this? Or would it suffice to change the existing variable?